recipes/sensormapgeo/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "sensormapgeo" %}
+{% set version = "0.4.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sensormapgeo-{{ version }}.tar.gz
+  sha256: 4f435416cb98602882e0a122a5499b8ffb027a36b6eae5529e493ae235a5cfcd
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - gdal
+    - numpy
+    - _openmp_mutex =* =*llvm*  # [unix]
```

Comment:
See https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp

Reply:
Thanks for the hint. So is fine to use the `_openmp_mutex=*=*_llvm` line in the run requirements sections or should I better add `- libgomp      # [linux]` to the build section?

Reply:
Just
```
# meta.yaml
requirements:
  build:
    - llvm-openmp  # [osx]
    - libgomp      # [linux]
```